diff --git a/PyConf/python/PyConf/components.py b/PyConf/python/PyConf/components.py
index 98a3612da..9a0697382 100644
--- a/PyConf/python/PyConf/components.py
+++ b/PyConf/python/PyConf/components.py
@@ -28,7 +28,10 @@ from functools import lru_cache
 from html import escape as html_escape
 
 import GaudiKernel
-import pydot
+try:
+    import pydot
+except ImportError:
+    import pydotplus as pydot
 
 from . import ConfigurationError
 from .dataflow import (
diff --git a/PyConf/python/PyConf/control_flow.py b/PyConf/python/PyConf/control_flow.py
index 42249c551..8feda4f11 100644
--- a/PyConf/python/PyConf/control_flow.py
+++ b/PyConf/python/PyConf/control_flow.py
@@ -126,7 +126,10 @@ except ImportError:
     from cgi import escape as html_escape
 
 from enum import Enum
-import pydot
+try:
+    import pydot
+except ImportError:
+    import pydotplus as pydot
 
 from .components import Algorithm
 from .dataflow import DataHandle
